react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js

```diff
@@ -102,11 +102,25 @@ class ChromeExtensionBanner extends PureComponent<Props, State> {
      *
      * @inheritdoc
      */
-    async componentDidUpdate() {
+    async componentDidUpdate(prevProps) {
         if (!this._isSupportedEnvironment()) {
             return;
         }
 
+        if (
+            this.props.chromeExtensionUrl
+            && !prevProps.chromeExtensionUrl
```

Comment:
doesn't this fit in one line? note that our eslint config checks for 120 line lengths

Reply:
done